_test/test_mpi/test_job_dispatcher_parpool.m

```diff
@@ -98,12 +98,15 @@ function create_test_file(name)
             assertTrue(exist(file3,'file') == 2);
             delete(cjob)
         end
-        % DISABLED
+        % DISABLED on Jenkins
         function test_job_fail_restart(obj, varargin)
-            warning('test_job_dispatcher_parpool:test_job_fail_restart disabled')
-            return;
-            % disp('test_job_dispatcher_parpool:test_job_fail_restart')
-            % test_job_fail_restart@job_dispatcher_common_tests(obj, varargin{:})
+             if is_jenkins
+                warning('test_job_dispatcher_parpool:test_job_fail_restart disabled')
+                return;
+            else
```

Comment:
no need for an `else` after `if xxx; return`

Reply:
ok, though not essential. Fixed as part  of group